diff --git a/fserver.defaults.inc b/fserver.defaults.inc
index 04a6afe..8d31437 100644
--- a/fserver.defaults.inc
+++ b/fserver.defaults.inc
@@ -329,10 +329,19 @@ function _fserver_content_default_fields() {
     'active' => '1',
     'referenceable_types' => array(
       'fserver_project' => 'fserver_project',
-      'image' => 0,
+      'blog' => 0,
+      'book' => 0,
+      'casetracker_basic_case' => 0,
+      'event' => 0,
+      'group' => 0,
+      'profile' => 0,
+      'casetracker_basic_project' => 0,
       'fserver_release' => 0,
+      'shoutbox' => 0,
+      'feed_ical_item' => 0,
+      'feed_ical' => 0,
     ),
-    'advanced_view' => '--',
+    'advanced_view' => 'fserver_projects',
     'advanced_view_args' => '',
     'widget' => array(
       'autocomplete_match' => 'contains',
diff --git a/fserver.features.node.inc b/fserver.features.node.inc
index 514ef68..7c46fe8 100644
--- a/fserver.features.node.inc
+++ b/fserver.features.node.inc
@@ -6,7 +6,7 @@
 function _fserver_node_info() {
   $items = array(
     'fserver_project' => array(
-      'name' => t('Project'),
+      'name' => t('Feature'),
       'module' => 'features',
       'description' => t('A feature project page. Provides a summary of project information and releases for download.'),
       'has_title' => '1',
diff --git a/fserver.info b/fserver.info
index b005a51..08cebe2 100644
--- a/fserver.info
+++ b/fserver.info
@@ -20,11 +20,12 @@ features[content][] = "fserver_release-field_fserver_versionextra"
 features[content][] = "fserver_release-field_fserver_versionmajor"
 features[content][] = "fserver_release-field_fserver_versionpatch"
 features[context][] = "fserver-section-project"
-features[menu][] = "fserver"
 features[node][] = "fserver_project"
 features[node][] = "fserver_release"
 features[views][] = "fserver_projects"
 features[views][] = "fserver_releases"
 features[views][] = "fserver_updates"
+spaces[types][] = "og"
+spaces[types][] = "site"
 name = "Feature Server"
 package = "Features"
